hooks/build

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Remove build file

Reply:
Probably need @kjrush to double check this. I know this was related to docker auto builds which he just reviewed.

Reply:
@kjrush can we just leave this hooks/build as it is?

Reply:
Yeah, I think we need to leave this in place, this is the step that ensures we get metadata in our docker images like the version ID.